packages/application-shell/src/components/navbar/menu-items.styles.ts

```diff
@@ -110,14 +112,227 @@ const NavlinkClickableContent = styled.div`
   width: 100%;
 `;
 
+const listStyles = css`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  min-height: 0;
+  flex: 1 1 0;
+`;
+
+const sublistStyles = css`
+  padding: var(--spacing-30);
```

Comment:
Since we're moving from the CSS modules world to the CSS-in-JS one, I think it would make sense to use the design tokens object instead of CSS vars: 
1. we would keep consistency with the rest of `app-kit` components
2. it would be easier to maintain as they are easily navigable in the editor and even more discoverable

```
padding: ${designTokens.spacing30}
```

Reply:
Great idea 🙌 Updated in f8e8dbc7693a881ea74c2ee71da5c721da11d49c